lib/rules/space-infix-ops.js

```diff
@@ -18,7 +18,7 @@ module.exports = function(context) {
         "?", ":", ","
     ];
 
-    function isSpaced(left, right) {
+    function firstNonSpacedNode(left, right) {
```

Comment:
Dang it. I had that in my mind but forgot to put it.
Will make the change. Thanks